azure-pipelines.yml

```diff
@@ -30,13 +30,14 @@ trigger:
   branches:
     include:
     - master
+    - "*-maintenance"
 
 pr:
   drafts: false
   branches:
     include:
-      - master
-      - "*-maintenance"
+    - master
+    - "*-maintenance"
```

Comment:
I know this is working, but the correct way is indenting with two spaces the list inside the group? At least is how I used it until now but I'm not a Yaml expert...

Reply:
Me neither - the rest of `azure-pipelines.yml` seems to be placing list elements on the same level as the list name, so this is just making the files consistent.
If it is wrong it should be amended for the entire file in a separate pull request.

Reply:
I've made some tests in some online yaml validators, and it seems they accept both forms, so this seems ok...

Reply:
Yeah, YAML is a lot like JS - unless it's obviously wrong it's assumed to be right.